lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
+          if self.class.minimum_approvals == nil or self.class.minimum_approvals.to_i == 0
```

Comment:
`self.class.minimum_approvals == nil`
can be
`self.class.minimum_approvals.nil?`